fuel/converters/svhn.py

```diff
@@ -290,11 +290,11 @@ def convert_svhn_format_2(directory, output_directory,
 
     train_set = loadmat(os.path.join(directory, FORMAT_2_TRAIN_FILE))
     train_features = train_set['X'].transpose(3, 2, 0, 1)
-    train_targets = train_set['y']
+    train_targets = train_set['y'] - 1
```

Comment:
What we need is for `10` to be mapped to `0` while other labels are untouched. In this case, you map `1` to `0`, `2` to `1`, etc.